frontend/src/App.js

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import LayoutGrid from "./components/LayoutGrid";
+
+
+
+export default function App() {
+    return (
+        <div className="App">
+    <LayoutGrid/>
```

Comment:
```suggestion
    <LayoutGrid/>
```
This name is a little bit confusing on the upper layer